machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
```

Comment:
I second all the extra suggestions, though some caution: the extra powercfg commands (`SUB_*`) modify whichever scheme is active. When we implemented similar functionality for the native compiler perf tests, we created a new scheme as a clone of `SCHEME_MIN` and modified that one; it might be worth doing that here as well, unless we know these machines are going to be used for perf only and don't mind leaving them in peak performance mode (incl. no sleep and video always on) 24/7.